spec/descriptions/tagSyntheticSettings.md

```diff
@@ -26,19 +26,22 @@ The API endpoints of this group can be used to manage Synthetic Locations and Sy
-      WebpageScript, and DNSAction. The locations assigned to execute this Synthetic
+    - **syntheticType** The type of the Synthetic test. Supported values are HTTPAction, HTTScript, BrowserScript (Beta), WebpageAction (Beta),
+      WebpageScript (Beta), and DNSAction (Not supported yet). The locations assigned to execute this Synthetic
       test must support this syntheticType, i.e. the location's playbackCapabilities property.
     - **markSyntheticCall** Flag used to control if HTTP calls will be marked as synthetic calls/endpoints in Instana backend, so they can be ignored when calculating service and application KPIs, users can also check "Hide Synthetic Calls" checkbox to hide/show them in UI.
     - **retries** An integer type from 0 to 2, 0 by default.
       It indicates how many attempts (max 2) will be allowed
       to get a successful connection (not necessarily a successful result).
       Failures like socket hangups, gateway timeouts, and DNS lookup fails cause retires, but 404's 400's, do not.
     - **retryInterval** The time interval between retries in seconds. The default is 1s, max is 10s.
-    - **timeout** The timeout to be used by the PoP playback engines running the test. Values in integer followed by time unit (ms, s, m). If timeout is not provided the playback engine will use its own timeout value. The default timeout is **1m** in PoP playback engines for HTTPAction and HTTPScript tests if user does not specify it. If user defined timeout value exceeds the `maxTimeout` configured in PoP deployment or `testFrequency` in test configuration, the timeout value does not take effect and PoP playback engines use the smaller one of `maxTimeout` and `testFrequency` as the actual timeout value.
+    - **timeout** The timeout to be used by the PoP playback engines running the test. Values in integer followed by time unit (ms, s, m). 
+      If timeout is not provided the playback engine will use its own timeout value. The default timeout is **1m** in PoP playback engines for HTTPAction and HTTPScript tests,
+      and **5m** for BrowserScript, WebpageAction, and WebpageScript tests if user does not specify it. 
+      If user defined timeout value exceeds the `maxTimeout` configured in PoP deployment or `testFrequency` in test configuration, 
+      the timeout value does not take effect and PoP playback engines use the smaller one of `maxTimeout` and `testFrequency` as the actual timeout value.
     - **XXXConfiguration** The configuration corresponding to the syntheticType. Configuration types are HTTPActionConfiguration, HTTPScriptConfiguration,
-      BrowserScriptConfiguration, WebpageActionConfiguration, WebpageScriptConfiguration, and DNSActionConfiguration. Right now, only HTTPActionConfiguration
-      and HTTPScriptConfiguration are supported.
+      BrowserScriptConfiguration (Beta), WebpageActionConfiguration (Beta), WebpageScriptConfiguration (Beta), and DNSActionConfiguration. 
```

Comment:
DNSActionConfiguration not supported

Reply:
Have removed DNSActionConfiguration from this file.